### ISSUE TITLE
Carby V2 gcplug firmware flashed but no video output

### ISSUE THREAD

**Issue by Sirethief2525** [ikorb/gcvideo#50]
I'm having this issue with my Carby device when I use the gcplug firmware it flashes successfully but no video shows on the screen, but when I use the shuriken firmware it works for the exception I have no access to the gcvideo menu to run the update proper. Any information you can give me on the issue would be a great help, thank-you in advance.

**Comment by ikorb** [comment]
I have no idea what firmware Carby V2 is supposed use, your vendor should be able to advise you.

If it seems to work with the shuriken version, it is likely that it is the correct one. You might need to configure the IR remote codes again though - there should be a button on your device, holding it down for a few seconds should show a screen for that. Please remember to save the configuration in the main menu after configuring the IR codes so you don't have to do it again after turning off the console.

**Comment by Sirethief2525** [comment]
The only issue I have right now is that no matter how long I press the button to reconfigure the remote nothing happens, so I'm at a loss on that for the time being. I made a backup of the original firmware that was on it, so I can always re-flash that back on it, it's just kind of frustrating that after following the guide on the Carby website didn't seem to work for me. I did download their update file and ran it through xilinx to try to flash it but the option never appeared until I added one of the firmware files from your github. I did order a replacement that already has version 3 on it, so I suppose I can experiment with this one to find out what the problem could be, Thanks for your help and if you happen to find out anything else about my situation just let me know.
